packages/spruce-utils/lib/graphql.ts

```diff
@@ -91,30 +97,40 @@ export class GraphQLClient {
 				}
 			})
 
+			// @ts-ignore
```

Comment:
@kengoldfarb `wsLink.subscriptionClient` is a private field, so TS doesn't like this. Just ignoring that one, but if there's a better public method that'd be a good follow-up PR

Reply:
Hmm, that makes me think maybe we're doing it wrong...I'm not seeing any open issues related to that TS error.

But agreed it makes sense as a separate PR